hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java

```diff
@@ -1052,7 +1052,6 @@ private synchronized boolean waitForWork() {
           try {
             wait(timeout);
           } catch (InterruptedException e) {
-            LOG.info("Interrupted while waiting to retrieve RPC response.", e);
```

Comment:
Can change to debug, rather than removing?

Reply:
That could work, or may even set it to trace level logging. If this code path is expected to happen often then perhaps we only need it logged at trace. I am fine with either trace or debug plus removing the stack trace.

Reply:
Done. Thanks